components/com_users/src/Controller/UserController.php

```diff
@@ -164,7 +168,10 @@ public function logout()
         }
 
         // Show a message when a user is logged out.
-        $app->enqueueMessage(Text::_('COM_USERS_FRONTEND_LOGOUT_SUCCESS'), 'message');
+        if (ComponentHelper::getParams('com_users')->get('frontend_message', 0))
+        {
```

Comment:
```suggestion
        if (ComponentHelper::getParams('com_users')->get('frontend_message', 0)) {
```
Please use PSR-12 code style. Currently PHPCS fails in drone because of this.